src/overlays/actors/ovl_Demo_Effect/z_demo_effect.c

```diff
@@ -2042,7 +2042,7 @@ void DemoEffect_DrawGetItem(Actor* thisx, GlobalContext* globalCtx) {
 /**
  * Callback for the SkelCurve system to draw the animated limbs.
  */
-s32 DemoEffect_DrawTimewarpLimbs(GlobalContext* globalCtx, SkelAnimeCurve* skelCuve, s32 limbIndex, void* thisx) {
+s32 DemoEffect_DrawTimewarpLimbs(GlobalContext* globalCtx, SkelCurve* skelCurve, s32 limbIndex, void* thisx) {
```

Comment:
Since this is an overrideLimbDraw, shouldn't this be named to something a bit more appropriate?